<launch>

   <param name="/use_sim_time" value="true"/>


<!--  ************** add odom tf  ***************  -->
 <node pkg="p2_core" type="odom_tf" respawn="false" name="odom_tf" output="screen"/>
   


<!-- **************  scan_merger **************  -->
  <include file="$(find p2_core)/launch/scan_merger.launch"/>
  
<!-- **************  gmapping config **************  -->  
  <include file="$(find p2_core)/config/gmapping.launch.xml"/>




</launch>

